stix_shifter_modules/sentinelone/stix_transmission/results_connector.py

```diff
@@ -0,0 +1,197 @@
+                    item["registryPath"] = None
+                    item["registryKeyPath"] = None
+                if item.get("registryOldValue") is not None:
+                    oldvalue=item.get("registryOldValue")
+                    valuetype = item.get("registryOldValueType")
+                    if valuetype == "BINARY":
+                        valuetype = "REG_BINARY"
+                        item["registryOldValueType"] = valuetype
+                    registryOldValue = {"data": oldvalue, "data_type": valuetype}
+                    item["registryOldValue"] = registryOldValue
+                if item.get("registryValue") is not None:
+                    rvalue = item.get("registryValue")
+                    valuetype = item.get("registryOldValueType")
+                    if valuetype == "BINARY":
+                        valuetype = "REG_BINARY"
+                        item["registryOldValueType"] = valuetype
+                    registryValue = {"data": rvalue, "data_type": valuetype}
+                    item["registryValue"] = registryValue
+
+            if item.get("loginIsAdministratorEquivalent") is not None:
+                if item.get("loginIsAdministratorEquivalent") == "True" or \
+                        item.get("loginIsAdministratorEquivalent") == "TRUE":
+                    item["loginIsAdministratorEquivalent"] = True
+                elif item.get("loginIsAdministratorEquivalent") == "False" or \
+                        item.get("loginIsAdministratorEquivalent") == "FALSE":
+                    item["loginIsAdministratorEquivalent"] = False
+
+            ResultsConnector.replace_escape_character(item)
+
+            empty_keys = [k for k, v in item.items() if v is None]
+            for key in empty_keys:
+                del item[key]
+        return res_dict
+
+    @staticmethod
+    def replace_escape_character(item):
+        """
+        replace escape character from results in json format
+        :param item: list of dictionary items
+        """
+        if item is not None:
+            fields = ['processCmd','srcProcCmdLine','tgtProcCmdLine','srcProcParentImagePath',
+                    'processImagePath','signatureSignedInvalidReason','srcProcActiveContentPath',
+                    'srcProcImageSha1', 'srcProcParentActiveContentPath','srcProcParentCmdLine',
+                    'srcProcParentDisplayName','srcProcParentImagePath','tgtFilePath',
+                    'user','srcProcUser','srcProcParentImageSha1', 'fileFullName',
+                    'storyline', 'tgtFileInternalName', 'tgtFileDescription',
+                    'publisher', 'siteName', 'srcProcImageSha256', 'srcProcParentImageSha256',
+                    'srcProcParentPublisher', 'srcProcParentStorylineId', 'srcProcParentUser',
+                    'srcProcPublisher','tgtFileOldMd5','gtFileOldPath','tgtFileOldSha1',
+                    'tgtFileOldSha256','tgtFileSha1','tgtFileSha256']
```

Comment:
why do you need escape the double quotes for this list of fields?

Reply:
In case of "srcProcCmdLine" when getting character (\") in between the string , it was causing parsing error when loading the stix json data. Hence removed this double quote (\") from that string value and it got resolved. Updated the other fields which might have similar data. 
example : 
"\"C:\\Users\\Administrator\\AppData\\Local\\GitHubDesktop\\app-2.9.6\\GitHubDesktop.exe\" --type=renderer --field-trial-handle=1592,6834333573192161485,838135359727459317,131072 --enable-features=WebComponentsV0Enabled --disable-features=CookiesWithoutSameSiteMustBeSecure,SameSiteByDefaultCookies,SpareRendererForSitePerProcess --lang=en-US --app-user-model-id=com.squirrel.GitHubDesktop.GitHubDesktop --app-path=\""C:\\Users\\Administrator\\AppData\\Local\\GitHubDesktop\\app-2.9.6\\resources\\app\"" --node-integration --no-sandbox --no-zygote --enable-remote-module --background-color=#fff --disable-blink-features=Auxclick --enable-spellcheck --enable-websql --disable-electron-site-instance-overrides --device-scale-factor=1.5 --num-raster-threads=1 --renderer-client-id=3 --no-v8-untrusted-code-mitigations --mojo-platform-channel-handle=1976 /prefetch:1"

Reply:
Now changed the code to escape it, hence replacing the '"' into '\\"'.